From 5101527fb425c2d17ef04b0ed87b3810cf6db7bc Mon Sep 17 00:00:00 2001
From: Cornelia Huck <cohuck@redhat.com>
Date: Wed, 26 May 2021 11:01:00 +0200
Subject: [PATCH] s390x/redhat: disable experimental 3270 device

RH-Author: Cornelia Huck <cohuck@redhat.com>
RH-MergeRequest: 9: Disable the 'x-terminal3270' device in qemu-kvm on s390x
RH-Commit: [1/1] 1df70e856fba434d06587a05c44c4dc5a367f1f7 (cohuck/qemu-kvm)
RH-Bugzilla: 1962479
RH-Acked-by: Thomas Huth <thuth@redhat.com>
RH-Acked-by: David Hildenbrand <david@redhat.com>
RH-Acked-by: Miroslav Rezanina <mrezanin@redhat.com>

The "x-terminal3270" device has been experimental for years,
has known limitations, has no known users anymore, and is not
supported by libvirt.

Therefore, disable it in our downstream builds, as it is not
really supported.

Signed-off-by: Cornelia Huck <cohuck@redhat.com>
---
 default-configs/devices/s390x-rh-devices.mak | 1 -
 1 file changed, 1 deletion(-)

diff --git a/default-configs/devices/s390x-rh-devices.mak b/default-configs/devices/s390x-rh-devices.mak
index 08a15f3e01..fe8613b8f6 100644
--- a/default-configs/devices/s390x-rh-devices.mak
+++ b/default-configs/devices/s390x-rh-devices.mak
@@ -6,7 +6,6 @@ CONFIG_S390_FLIC=y
 CONFIG_S390_FLIC_KVM=y
 CONFIG_SCLPCONSOLE=y
 CONFIG_SCSI=y
-CONFIG_TERMINAL3270=y
 CONFIG_VFIO=y
 CONFIG_VFIO_AP=y
 CONFIG_VFIO_CCW=y
-- 
2.27.0

